### PR TITLE
Guide for using with ethers.js

### PR DESCRIPTION
**Pull request by trizin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,6 +73,29 @@ const provider = await web3Modal.connect();
...
+  /* See Provider Options Section */
+};
+
+const web3Modal = new Web3Modal({
+  network: "mainnet", // optional
+  cacheProvider: true, // optional
+  providerOptions // required
+});
+
+const provider = await web3Modal.connect();
```
Comment (ochikov) [inline README.md:92]: I've named this `instance` because it is actually not a provider.

File: README.md [hunk 2]
```diff
@@ -73,6 +73,29 @@ const provider = await web3Modal.connect();
...
+
+const ethersProvider = new ethers.providers.Web3Provider(provider);
```
Comment (ochikov) [inline README.md:94]: And then this is just a provider.

**Comment by johnfrades** [comment]
Why this one is not merged?

**Comment by johnfrades** [comment]
Finally. Thank you

**Pull request merged**